tox.ini

```
@@ -1,17 +1,18 @@
 [tox]
-envlist = py{37,38,39,310,311,312}-test-sphinx{40,53,62,70,71,dev}{-cov}{-clocale}
+envlist = py{38,39,310,311,312}-test-sphinx{40,53,62,70,71,72,dev}{-cov}{-clocale}
```

Would have to do this too, I guess.
```suggestion
envlist = py{38,39,310,311,312}-test-sphinx{_oldest,53,62,70,71,72,dev}{-cov}{-clocale}
```